xbmc/platform/android/activity/AndroidKey.cpp

```diff
@@ -264,44 +264,44 @@ bool CAndroidKey::onKeyboardEvent(AInputEvent *event)
   {
     case AKEY_EVENT_ACTION_DOWN:
 #if 1
-      CXBMCApp::android_printf("CAndroidKey: key down (dev:%d; src:%d; code: %d; repeat: %d; flags: 0x%0X; alt: %s; shift: %s; sym: %s)",
+      /*CXBMCApp::android_printf("CAndroidKey: key down (dev:%d; src:%d; code: %d; repeat: %d; flags: 0x%0X; alt: %s; shift: %s; sym: %s)",
         deviceId, source, keycode, repeat, flags,
         (state & AMETA_ALT_ON) ? "yes" : "no",
         (state & AMETA_SHIFT_ON) ? "yes" : "no",
-        (state & AMETA_SYM_ON) ? "yes" : "no");
+        (state & AMETA_SYM_ON) ? "yes" : "no");*/
 #endif
       XBMC_Key((uint8_t)keycode, sym, modifiers, unicode, false);
       break;
 
     case AKEY_EVENT_ACTION_UP:
 #if 1
-      CXBMCApp::android_printf("CAndroidKey: key up (dev:%d; src:%d; code: %d; repeat: %d; flags: 0x%0X; alt: %s; shift: %s; sym: %s)",
+      /*CXBMCApp::android_printf("CAndroidKey: key up (dev:%d; src:%d; code: %d; repeat: %d; flags: 0x%0X; alt: %s; shift: %s; sym: %s)",
         deviceId, source, keycode, repeat, flags,
         (state & AMETA_ALT_ON) ? "yes" : "no",
         (state & AMETA_SHIFT_ON) ? "yes" : "no",
-        (state & AMETA_SYM_ON) ? "yes" : "no");
+        (state & AMETA_SYM_ON) ? "yes" : "no");*/
 #endif
       XBMC_Key((uint8_t)keycode, sym, modifiers, unicode, true);
       break;
 
     case AKEY_EVENT_ACTION_MULTIPLE:
 #if 1
-      CXBMCApp::android_printf("CAndroidKey: key multiple (dev:%d; src:%d; code: %d; repeat: %d; flags: 0x%0X; alt: %s; shift: %s; sym: %s)",
+      /*CXBMCApp::android_printf("CAndroidKey: key multiple (dev:%d; src:%d; code: %d; repeat: %d; flags: 0x%0X; alt: %s; shift: %s; sym: %s)",
         deviceId, source, keycode, repeat, flags,
         (state & AMETA_ALT_ON) ? "yes" : "no",
         (state & AMETA_SHIFT_ON) ? "yes" : "no",
-        (state & AMETA_SYM_ON) ? "yes" : "no");
+        (state & AMETA_SYM_ON) ? "yes" : "no");*/
 #endif
       return false;
       break;
 
     default:
 #if 1
-      CXBMCApp::android_printf("CAndroidKey: unknown key (dev:%d; src:%d; code: %d; repeat: %d; flags: 0x%0X; alt: %s; shift: %s; sym: %s)",
+      /*CXBMCApp::android_printf("CAndroidKey: unknown key (dev:%d; src:%d; code: %d; repeat: %d; flags: 0x%0X; alt: %s; shift: %s; sym: %s)",
         deviceId, source, keycode, repeat, flags,
         (state & AMETA_ALT_ON) ? "yes" : "no",
         (state & AMETA_SHIFT_ON) ? "yes" : "no",
-        (state & AMETA_SYM_ON) ? "yes" : "no");
+        (state & AMETA_SYM_ON) ? "yes" : "no");*/
```

Comment:
if you are going this route, its probably better to just change the if def to #if 0 for each of these switch cases

Reply:
I'm also fine with changing the #if to 0